app/components/ui/create-role-modal.tsx

```diff
@@ -0,0 +1,96 @@
+import { Form } from "@remix-run/react";
+import { FunctionComponent } from "react";
+
+import { Button } from "~/components/ui/button";
+
+export type ModalsType = {
+  className?: string;
+  onClose?: () => void;
+};
+
+const CreateRoleModal: FunctionComponent<ModalsType> = ({
+  className = "",
+  onClose,
+}) => {
+  return (
+    <div
+      role="dialog"
+      aria-modal="true"
+      aria-labelledby="modal-title"
+      className={`m-auto flex min-h-[435px] max-w-[465px] flex-col rounded-md bg-zinc-50 px-6 py-6 shadow-lg ${className}`}
+    >
```

Comment:
Please use the ShadCN modal instead of this custom implementation.

Reply:
Alright

Reply:
Done.

Reply:
You say done but I don't see anything